test/cql-pytest/test_schema.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Tests for schema related things
+
+import time
+import pytest
+
+from util import new_test_table
+from nodetool import flush
+
+
+def testDropColumnWithTimestampPrepared(scylla_only, cql, test_keyspace):
```

Comment:
I would have added a comment explaining explicitly (so the reader won't have to guess...) that Cassandra does NOT allow "USING TIMESTAMP ?" and we decided to allow it - so this test checks only the prepared case (and we have other tests in cassandra_tests/ that test the unprepared case).
As I noted above, this also brings the question why we decided to support this case that Cassandra doesn't. This question should be answered somewhere in the commit message, or in the comment on this case - but we need to realize that it was a deliberate decision, and why we took it. Because I'd say that 80% of this series deals with all the fallout from this specific decision.

Reply:
Also please add a comment saying which issue this test reproduces.